### PR TITLE
Removed getName() method from TableBundleExtensionInterface

### PR DESCRIPTION
**Pull request by senaria**
- removed getName() from TableBundleExtensionInterface
- removed  AbstractTableBundleExtension 
- removed getName() from BuildInFunctionExtension which implements  now TableBundleExtensionInterface
- renamed  TableBundleExtensionInterface to UqlExtensionInterface
- renamed and relocated other files inside the Extension folder

### CODE REVIEW

File: Extension/AbstractTableBundleExtension.php [hunk 1]
```diff
@@ -4,8 +4,6 @@
 
 abstract class AbstractTableBundleExtension implements TableBundleExtensionInterface
```
Comment (yakobe) [inline Extension/AbstractTableBundleExtension.php]: Table bundle here? That's a different component? It appears some of the dependencies are messed up or there is a naming issue. Can you investigate and then we can discuss?

Comment (senaria) [inline Extension/AbstractTableBundleExtension.php]: Looks like that the naming is a little bit mixed up in here. This thing is in the end an AbstractUqlExtension which makes its containing UqlFunctions available inside the UQL language.

File: Resources/config/services.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ services:
             - { name: netdudes_data_sourcery.extension }
 
     netdudes_data_sourcery.extension_container:
-        class: Netdudes\DataSourceryBundle\Extension\TableBundleExtensionContainer
+        class: Netdudes\DataSourceryBundle\Extension\UqlExtensionContainer
```
Comment (yakobe) [inline Resources/config/services.yml]: Should this class be called just `ExtensionContainer`? Or maybe we should call the service something like `netdudes_data_sourcery.uql_extension_container` or similar? What do you think?

Comment (senaria) [inline Resources/config/services.yml]: Service name can be changed. The class I would leave as it is.

**Comment by yakobe** [comment]
:shipit:

**Pull request merged**